docs/README-EDITING.md

```diff
@@ -66,77 +46,128 @@ sitemap:
+## Documentation versioning
 
-In a browser please go into: `/docs/[number]`.
+To create a new version of the Handsontable documentation:
 
-## Editing API Reference
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
+```
 
-To edit API Reference goes into source code and change jsdocs comments. Then, to generate API Reference pages:
+To remove an existing version of the Handsontable documentation:
 
-```shell script
-npm run docs:api
+* Remove the required version's [directory](./README.md#handsontable-docs-directory-structure):
+```bash
+rm -rf ./<semver.version>
 ```
 
-## Versioning
+## Markdown containers
+
+To render content in different ways, the documentation uses custom Markdown containers, for example:
+
+```markdown
+::: example #exampleId .class :preset --html 1 --js 2
 
-**Release a version**
+// code example
 
-```shell script
-npm run docs:version
+:::
 ```
 
-Then you'll be asked for the version that you want to release (interactive mode).
+We use the following Markdown containers:
 
-**Remove a version**
+| Container                    | Usage                                                      |
+|------------------------------|------------------------------------------------------------|
+| `::: tip`                    | A note.                                                    |
+| `::: example [options]`      | Renders a code example as specified in [options].          |
+| `::: source-code-link [URL]` | Adds a source code link to the right of an API ref header. |
 
-It is enough to remove version directory.
+### Adding code examples
+Using the `example` Markdown container, you can add code snippets that show the code's result:
 
-```shell script
-rm -rf ./[NUMBER]
+```js
+::: example #exampleId .class :preset --html 1 --js 2 --css 3  --no-edit --tab preview
+// `#example1` is the example's ID, for creating a Handsontable container
+
+    ```js
+    // code here
+    ```
+:::
 ```
+
+The `example` Markdown container offers the following options:
+
+| Option           | Required | Example           | Possible values                                            | Usage                          |
+|------------------|----------|-------------------|------------------------------------------------------------|--------------------------------|
+| `#exampleId`     | Yes      | `#data-grid-1`    | String                                                     | Container's unique ID.         |
+| `.class`         | No       | `.new-class`      | String                                                     | Container's custom class.      |
+| `:preset`        | No       | `:react`          | [`preset`](.vuepress/handsontable-manager/dependencies.js) | Sets code dependencies.        |
+| `--codeOption 1` | Yes      | `--html 1 --js 2` | [`codeOption`](.vuepress/highlight.js) + Number            | Sets code snippets' positions. |
```

Comment:
Could you divide it to separated position?

| `--js <pos>` | No | `--js 1` | Positive Integer, default 1 | Sets the position of code snippet in a markdown container |
| `--html [pos]` | No | `--html 2` | Positive Integer, default 0 (0=disabled)  | Sets the position of html snippet in a markdown container. The 0 value disables the HTML tab |
| `--css [pos]` | No | `--css 2` | Positive Integer, default 0 (0=disabled)  | Sets the position of css snippet in a markdown container. The 0 value disables the CSS tab |